src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -217,6 +217,10 @@
     "NAME": "Feast",
     "DESCRIPTION": "Remember stsjorbsmod:Gluttony. NL Draw !M! cards. NL Exhaust."
   },
+  "stsjorbsmod:Feint": {
+    "NAME": "Feint",
+    "DESCRIPTION": "Deal !D! Damage. NL Draw 2 Cards. NL Discard 1 Card."
```

Comment:
We prefer to use placeholders for numbers on cards rather than hardcoding them into the description strings; even if they don't ever change in-game (like these ones), numbers frequently change during later balancing passes, and using placeholders from the start means fewer changes to all the translations' strings files for simple balance changes.

Since these effects aren't block/damage, you would want to use "magic number" placeholders for these 2 effects. For example, you might do something like

`"Deal !D! damage. NL Draw !M! cards. NL Discard !stsjorbsmod:MetaMagic! card."`

...and then fill in the placeholders by adding lines to the constructor for the card implementation:

```java
baseMagicNumber = magicNumber = DRAW;
baseMetaMagicNumber = metaMagicNumber = DISCARD;
```